plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/AppEngineProjectConfigTest.java

```diff
@@ -34,35 +34,37 @@
 public class AppEngineProjectConfigTest {
 
   private AppEngineProjectConfig config = new AppEngineProjectConfig();
-  
+
+  @Test
   public void testProject() throws CoreException {
     try {
       IWorkspace workspace = ResourcesPlugin.getWorkspace();
       IProject project = workspace.getRoot().getProject("foobar");
+      project.create(null);
       config.setProject(project);
-      Assert.assertTrue(project.getRawLocationURI().getPath().endsWith("foobar"));
+      Assert.assertTrue(project.getLocationURI().getPath().endsWith("foobar"));
```

Comment:
What's the difference between raw and cooked? Why prefer one over the other in this test?